openshift/template.yaml

```diff
@@ -461,6 +463,8 @@ objects:
                 value: ${INSTALLER_CACHE_CAPACITY}
               - name: ENABLE_OKD_SUPPORT
                 value: ${ENABLE_OKD_SUPPORT}
+              - name: OCP_MCE_RELEASE_MATRIX
```

Comment:
I think there are also thing that should be updated here: https://github.com/openshift/assisted-service/blob/master/deploy/assisted-service-configmap.yaml